docs/source/actions/merge.rst

```diff
@@ -171,6 +173,21 @@ request:
 Defining the Commit Message
 ---------------------------
 
+|deprecated tag|
+
+.. warning::
+
+    ``commit_message`` is deprecated and will be removed on April 25th, 2022. It's replaced by :ref:`commit_message_template <commit_message_template>`.
+    To migrate from ``commit_message: title+body`` to the :ref:`commit_message_template <commit_message_template>` attribute, set it to:
+
+   .. code-block:: md
+   
+    ## Commit Message Template
```

Comment:
why this here? 🤔

Reply:
To put the code block in the warning i indent it, i think it's prettier like that, no ?

Reply:
I'm not talking about the indentation, but the title here. Why would you need a title here?
Why would you set a markdown title in `commit_message_template`?

Reply:
I wanted to do like the ## Commit Message, I thought it was like a comment with the #, so that the user understands which part it corresponds to. It is not used for that ?

Reply:
`#` is used in some language for commenting, but I don't know any language using `##` as a comment delimiter. 

As you might know, `commit_message_template` is a Jinja2 template that generates text for a commit message.
Here's how to put comments in Jinja2: https://jinja.palletsprojects.com/en/3.0.x/templates/#comments

So in the end, `## Commit Message Template` is going to be the **title** of the commit with your example. I doubt anyone would want that 😅

Reply:
Okay, i thought it was a specific syntax for documentation, i'll put that in {%%}

Reply:
in fact i'm just going to take it off, as the user will not see it